tests/translate.service.spec.ts

```diff
@@ -296,4 +296,19 @@ describe('TranslateService', () => {
         expect(browserCultureLand).toBeDefined();
         expect(typeof browserCultureLand === 'string').toBeTruthy();
     });
+
+    it('should not make duplicate requests', fakeAsync(() => {
+        let getTranslationCalls = 0;
+        spyOn(translate.currentLoader, 'getTranslation');
+        translate.currentLoader.getTranslation['and'].callFake(() => {
```

Comment:
This is throwing a TS compilation error: `TS7017: Index signature of object type implicitly has an 'any' type.`

Looking for help on how to get rid of this. `.and` does not work either.